lib/plugins/processor/asset.js

```diff
@@ -7,7 +7,7 @@ const { extname, relative } = require('path');
 const { Pattern } = require('hexo-util');
 
 module.exports = ctx => {
-  function processPage(file) {
+  async function processPage(file) {
```

Comment:
Avoid using `async` here. The generation performance has been dropped by 30% after the PR.